l10n_es_aeat/__manifest__.py

```diff
@@ -22,7 +22,7 @@
     "development_status": "Mature",
     "depends": ["l10n_es", "account_tax_balance"],
     # odoo_test_helper is needed for the tests
-    "external_dependencies": {"python": ["unidecode", "cryptography>=3"]},
+    "external_dependencies": {"python": ["unidecode", "cryptography", "packaging"]},
```

Comment:
No se puede evitar la dependencia de `packaging` por no introducir dependencias extra?

Reply:
We can use tuple(map(int, cryptography.__version__.split("."))) >= (3, 0)
But is more clear use package